core/pkg/resultshandling/printer/v2/junit.go

```diff
@@ -118,96 +119,119 @@ func (junitPrinter *JunitPrinter) ActionPrint(opaSessionObj *cautils.OPASessionO
 func testsSuites(results *cautils.OPASessionObj) *JUnitTestSuites {
 	return &JUnitTestSuites{
 		Suites:   listTestsSuite(results),
-		Tests:    results.Report.SummaryDetails.NumberOfControls().All(),
+		Tests:    results.Report.SummaryDetails.NumberOfResources().All(),
```

Comment:
The correct numbers should be the number of files... But for now, we should not change it.